### PR TITLE
Create Request Model

### PR DESCRIPTION
**Pull request by nicholasrwx**
### What this PR does (required):
- This PR address the creation of the `Request Model` as per #9
- its the Model for database Queries Via the `request` routes
- This is for fetching all requests associated with the logged in user, creating new requests, and updating existing requests.

### Any information needed to test this feature (required):
- To test this feature you would have to run the  request `API Calls` in the `client/helpers/APICalls` folder, with test data, as they are not connect to any form or button in the app yet.

### CODE REVIEW

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Please make those changes and commit them for another review.

File: server/models/Request.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  sitter_id: {
+    type: String,
+  },
+  start: {
+    type: Number,
+  },
+  end: {
+    type: Number,
+  },
```
Comment (ezeilo-su) [inline server/models/Request.js:15]: These ones should be of type `Date` with a default value. Please refer to the issue.

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Please implement the requested changes for another review.

File: server/models/Request.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  accepted: {
+    type: Boolean,
+  },
+  declined: {
+    type: Boolean,
+  },
+  paid: {
+    type: Boolean,
+  },
```
Comment (ezeilo-su) [inline server/models/Request.js:24]: These ones should have a default value. Please refer to the issue.

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Make those changes, please. :+1:

**Comment by nicholasrwx** [comment]
I have addressed all the issues above, in this commit https://github.com/hatchways/team-igneous/pull/56/commits/615b2d25bf875cd32396ea2566226b9128d00f9d

**Comment by nicholasrwx** [comment]
I just added a couple extra values to the model earlier I didn't have a moment to push until now.

File: server/models/Request.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+  },
+  sitter_id: {
+    type: String,
+  },
```
[the remark this answers is not in the document]
Comment (nicholasrwx) [reply server/models/Request.js]: These changes have now been made and approved in https://github.com/hatchways/team-igneous/pull/56/commits/8ade7c03ebe6f587d3dc02b8f8aa85f5e9dcf490 :)

File: server/models/Request.js (hunk 1 above)
Comment (nicholasrwx) [reply server/models/Request.js]: These changes have now been made and approved in https://github.com/hatchways/team-igneous/pull/56/commits/8ade7c03ebe6f587d3dc02b8f8aa85f5e9dcf490 :)

File: server/models/Request.js (hunk 2 above)
Comment (nicholasrwx) [reply server/models/Request.js]: These changes have now been made and approved in https://github.com/hatchways/team-igneous/pull/56/commits/8ade7c03ebe6f587d3dc02b8f8aa85f5e9dcf490 :)

**Pull request merged**